feature/tabsflow/moreflow/faq/src/main/java/com/epmedu/animeal/tabs/more/faq/presentation/FAQScreenEvent.kt

```diff
@@ -0,0 +1,8 @@
+package com.epmedu.animeal.tabs.more.faq.presentation
+
+import com.epmedu.animeal.tabs.more.faq.presentation.model.FAQCard
+
+internal sealed interface FAQScreenEvent {
+    object BackClicked : FAQScreenEvent
```

Comment:
Because they have different directions. 
`FAQScreenEvent` - screen -> viewmodel
`FAQEvent` - viewmodel -> screen
Maybe we should create `ViewModelEvent` and `ScreenEvent` interfaces and apply them to `EventDelegate` and `handleEvents` to make the separation of concerns more obvious

Reply:
Anyway, removed events completely